reference/running.md

```diff
@@ -98,7 +98,21 @@ env:
   value: "true"
 ```
 
-With the Ambassador Edge Stack, if you set `AMBASSADOR_NAMESPACE` or `AMBASSADOR_SINGLE_NAMESPACE`, set it in deployment container. 
+With the Ambassador Edge Stack, if you set `AMBASSADOR_NAMESPACE` or `AMBASSADOR_SINGLE_NAMESPACE`, set it in deployment container.
+
+If you want to set a certificate for your `TLScontext` from another namespace, you can use the following:
+
+```yaml
+env:
+- name: AMBASSADOR_SINGLE_NAMESPACE
+  value: "YES"
+- name: AMBASSADOR_CERTS_SINGLE_NAMESPACE
```

Comment:
No `value:` property? It might work... I'm just not used to seeing this!

Reply:
That's the info the customer put in. @kflynn understands this a bit more than me however, so maybe he can chime in?

Reply:
I'd put `value: "YES"` like `AMBASSADOR_SINGLE_NAMESPACE` has.

Reply:
Okay, added value in. Please review again @kflynn